browser/brave_shields/cookie_expiry_browsertest.cc

```diff
@@ -114,6 +115,28 @@ class CookieExpirationTest : public InProcessBrowserTest {
   content::ContentMockCertVerifier mock_cert_verifier_;
 };
 
+// Test that a session cookie i.e. without a max-age or expiry date is not
+// affected by the cookie expiration policy. Also, is deleted on browser close.
```

Comment:
> Also, is deleted on browser close.

I don't see how the tests validates that.

Reply:
That's a leftover comment. I see now that you have a similar test though in browser/sessions/brave_session_restore_browsertest.cc, so IMO we don't need another one here.